services/src/atdd/src/test/java/org/openkilda/atdd/FlowIgnoreBandwidthTest.java

```diff
@@ -44,11 +46,11 @@ public void availableISLBandwidthsBetweenSwitches(String source, String dest, lo
             PathDto left = link.getPath().get(0);
             PathDto right = link.getPath().get(1);
 
-            if (! source.equals(left.getSwitchId())) {
+            if (!source.equals(left.getSwitchId())) {
```

Comment:
warning from IDEA:
```
'equals()' between objects of inconvertible types 'String' and 'SwitchId'
```
and is it right.

Reply:
I did not see this warning. I'll fix it.